packages/xod-client-electron/src/view/dialogUtils.js

```diff
@@ -0,0 +1,42 @@
+import os from 'os';
+import * as R from 'ramda';
+
+const addExtensionsIntoName = (fileFilter) => {
+  const extForName = R.ifElse(
+    R.isEmpty,
+    R.always('directory'),
+    R.concat('.')
+  )(fileFilter.extensions[0]);
+
+  return R.assoc(
+    'name',
+    `${fileFilter.name} (${extForName})`,
+    fileFilter
+  );
+};
+const transformDialogFileFilters = filters => R.compose(
+  fn => fn(filters),
+  R.cond([
+    [R.equals('darwin'), () => R.reverse],
+    [R.equals('linux'), () => R.map(addExtensionsIntoName)],
+    [R.T, () => R.identity],
+  ])
+)(os.platform());
+
+export const getSaveDialogFileFilters = () => transformDialogFileFilters([
+  { name: 'XOD Packed Project', extensions: ['xodball'] },
+  { name: 'XOD Multifile Project', extensions: ['', 'xod'] },
+]);
+export const getOpenDialogFileFilters = () => transformDialogFileFilters([
+  { name: 'Packed XOD project', extensions: ['xodball'] },
+  { name: 'Multifile XOD project', extensions: ['xod', 'xodp'] },
+]);
```

Comment:
Should names differ across open and save? I mean “XOD” word position. Better use a single style. I would recommend “XOD” in the second position since “Packed / Multifile” have more meaning, thus should go first.